.github/workflows/ci-daily-build.yml

```diff
@@ -0,0 +1,42 @@
+name: ci-ifcopenshell-daily
+
+on:
+  push:
+    paths:
+      - src/**
+      - conda/**
+      - .github/workflows/ci-daily-build.yml
+    branches:
+      - pr-daily-builds
```

Comment:
yes, when the other changes are given the "green light" I will remove this. Right now I keep it in place so that I can test the github actions pipeline on my fork